sdk/src/asset_handlers/c2pa_io.rs

```diff
@@ -60,6 +60,10 @@ impl AssetIO for C2paIO {
 
         Ok(vec![hop])
     }
+
+    fn remove_cai_store(&self, _asset_path: &Path) -> Result<()> {
+        Ok(())
```

Comment:
Seems like this should be some kind of error signaling a non-sensical request instead of `Ok(())`.

Reply:
Yeah, I am not sure about this one.  No one will ever call this so success or failure does not matter at this time.

Reply:
I don't want to return an error for a no op

Reply:
Agree with leaving this as `Ok(())`, but can we at least log that this was a non-sensical request?